src/Command/MeiliSearchClearCommand.php

```diff
@@ -2,16 +2,16 @@
 
 declare(strict_types=1);
 
-namespace MeiliSearch\Bundle\Command;
+namespace Meilisearch\Bundle\Command;
```

Comment:
The casing in the filenames of this class and the other ones in this folder should be changed too I suppose 🤔

Reply:
Yeah, I had an amazing time with git caching now, I cloned the repo twice and none of my file renames were taking place... anyway, it is fixed now!